common/scripts/update-imodeljs-native-version.sh

```diff
@@ -2,8 +2,9 @@
 
 usage() {
   echo "Updates the itwinjs-core repository to use a new version of the @bentley/imodeljs-native package."
-  echo "Usage: $0 -v VERSION"
+  echo "Usage: $0 -v VERSION -b BRANCH"
   echo "  -v VERSION: The semantic version (e.g., 3.2.7) of the @bentley/imodeljs-native package to which to update."
+  echo "  -b BRANCH: The branch on itwinjs-core repository (e.g., release/4.5.x) that the update is based off and checked against."
```

Comment:
maybe add that -b is optional and defaults to master if not included

Reply:
sure will do it now!

Reply:
I am done with that!